requirements.txt

```diff
@@ -1,6 +1,9 @@
+email-validator==1.1.2
 Flask==1.1.2
+Flask-Login==0.5.0
 flask_wtf==0.14.3
 mongoengine==0.23.0
 pylint==2.7.2
 python-dotenv==0.15.0
-pymongo==3.11.3
+Werkzeug==1.0.1
+WTForms==2.3.3
```

Comment:
@AndriiSkavreniuk add `email_validator` to requirements

Reply:
done